core/jobs/batch_jobs/firebase_validation_jobs.py

```
@@ -0,0 +1,148 @@
+
+from __future__ import annotations
+
+import dataclasses
+
+from core.jobs import base_jobs
+from core.jobs.io import firebase_io
+from core.jobs.types import firebase_adapters, job_run_result
+
+import apache_beam as beam
+from apache_beam import pvalue
+from typing import Iterable, TypedDict
```

Ruff (0.15.1)

[warning] Import from `collections.abc` instead: `Iterable`

Import from `collections.abc`

(UP035)